docs/develop/customize.md

```diff
@@ -22,3 +24,16 @@ Passing the `--no-install-js` option, skips re-installation of JS dependencies.
 Go to the browser [*https://localhost:5000/*](https://localhost:5000) or refresh the page. And voilà! The logo has been changed!
 
 **WARNING**: If you do not see it changing, check in an incognito window, the browser might have cached the logo.
+
+## Change colors
+
+Another common request that you might have is: *How do I change the colors so I can make my instance look like my institution's theme?*. We show what changes you need to do in order to achieve that.
+
+We are going to change the top header section in the frontpage to apply our custom background color. Open the `assets/scss/<your instance shortname>/variables.scss` file and edit it as below:
+
+``` console
+$navbar_background_image: unset;
+$navbar_background_color: #myColorCode;
+```
+
+Then, use the `update` command as above and refresh the page! You should be able to see your top header's color changed! You can find all the available variables that you can change [here](https://github.com/inveniosoftware/invenio-app-rdm/blob/master/invenio_app_rdm/theme/assets/scss/invenio_app_rdm/variables.scss).
```

Comment:
Doubt: Does it require stop-start of the instance? I would specify the command is from `invenio-cli`

Reply:
It doesn't require restart if you are in `local` mode. As long as, `FLASK_ENV=development` is there :) I am not sure about the `containerize` though, maybe @fenekku can elaborate more on this!

Reply:
For containerize, any change requires a re-run of `containerize`. 

The above is the right way to go for me: only mention running `update` because that's the flow we want developers to use. Perhaps, `invenio-cli update --no-install-js` to be more explicit.